src/ol3cesium.js

```diff
@@ -11,12 +11,26 @@ goog.require('olcs.VectorSynchronizer');
 
 
 /**
- * @param {!ol.Map} map
+ * @param {!(ol.Map|olcsx.OLCesiumOptions)} arg1 `ol.Map` or options
  * @param {Element|string=} opt_target Target element for the Cesium container.
+ *   Overrides a target set in options.
  * @constructor
  * @api
  */
-olcs.OLCesium = function(map, opt_target) {
+olcs.OLCesium = function(arg1, opt_target) {
```

Comment:
The old behaviour of taking a map and an optional target must be deprecated and will be removed in the near future. Additionally, a comment explaining how to switch to the new behaviour must be added.
We will later add a CHANGES.md file like in Cesium `https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md`.